modules/api/src/main/java/org/apache/ignite/catalog/annotations/Zone.java

```diff
@@ -57,11 +57,11 @@
     int replicas() default -1;
 
     /**
-     * Affinity function.
```

Comment:
There are still dozen of places where "affinity: is used, e.g.

-  modules/runner/README.MD "it calculates a new affinity for the table"
- org.apache.ignite.internal.sql.engine.trait.IgniteDistributions#affinity(int, int, java.lang.Object)

Is that intended?

Reply:
Here in PR it's removed :)
In README.MD it was actually missed, thank you. I removed this example where "affinity" word was, because it's hopelessly obsolete. Mayve we should revise the whole readme.

In IgniteDistribution and related classes and tests it's left intentionally, the word "affinity" has different meaning there and I created a separate ticket [IGNITE-23227](https://issues.apache.org/jira/browse/IGNITE-23227) to fix it.